src/PactNet/ConfiguredMessageVerifier.cs

```diff
@@ -80,9 +80,9 @@ public async Task VerifyAsync<T>(Func<T, Task> handler)
         private T MessageReified<T>()
         {
             string reified = this.server.Reify(this.message);
-            NativeMessage content = JsonConvert.DeserializeObject<NativeMessage>(reified);
+            NativeMessage content = JsonConvert.DeserializeObject<NativeMessage>(reified, config.DefaultJsonSettings);
```

Comment:
I don't think this line needs the settings adding. This is the response from the Rust FFI which wraps the message, not the actual message itself. It's the reified version below that needs the settings to be applied.